--- a/command.go
+++ b/command.go
@@ -41,6 +41,8 @@
 var CommandFuncs = map[parser.CommandType]CommandFunc{
 	token.BACKSPACE:   ExecuteKey(input.Backspace),
 	token.DELETE:      ExecuteKey(input.Delete),
+	token.END:         ExecuteKey(input.End),
+	token.HOME:        ExecuteKey(input.Home),
 	token.INSERT:      ExecuteKey(input.Insert),
 	token.DOWN:        ExecuteKey(input.ArrowDown),
 	token.ENTER:       ExecuteKey(input.Enter),
@@ -229,6 +231,10 @@
 			inputKey = &input.Space
 		case "Backspace":
 			inputKey = &input.Backspace
+		case "Home":
+			inputKey = &input.Home
+		case "End":
+			inputKey = &input.End
 		case "Left":
 			inputKey = &input.ArrowLeft
 		case "Right":
--- a/command_test.go
+++ b/command_test.go
@@ -13,7 +13,7 @@
 		t.Errorf("Expected %d commands, got %d", numberOfCommands, len(parser.CommandTypes))
 	}
 
-	const numberOfCommandFuncs = 31
+	const numberOfCommandFuncs = 33
 	if len(CommandFuncs) != numberOfCommandFuncs {
 		t.Errorf("Expected %d commands, got %d", numberOfCommandFuncs, len(CommandFuncs))
 	}
--- a/parser/parser.go
+++ b/parser/parser.go
@@ -321,6 +321,8 @@
 		case peek.Type == token.ENTER,
 			peek.Type == token.SPACE,
 			peek.Type == token.BACKSPACE,
+			peek.Type == token.HOME,
+			peek.Type == token.END,
 			peek.Type == token.MINUS,
 			peek.Type == token.AT,
 			peek.Type == token.LEFT_BRACKET,
